apps/api/scripts/seed_models.py

```
@@ -43,7 +43,10 @@
 sys.path.insert(0, str(backend_dir))
 
 # Import app modules after path setup  # noqa: E402
-from app.db.mongodb.collections import ai_models_collection, users_collection  # noqa: E402
+from app.db.mongodb.collections import (  # noqa: E402
```

Ruff (0.14.14)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)